cmd/registry/controller/controller.go

```diff
@@ -34,7 +33,7 @@ type Action struct {
 
 func ProcessManifest(
 	ctx context.Context,
-	client connection.RegistryClient,
+	listerClient Lister,
```

Comment:
to reduce confusion, could "listerClient" be simply "lister"?

Reply:
Renamed to client.